lib/ansible/modules/cloud/alicloud/ali_rds_instance.py

```diff
@@ -28,306 +28,299 @@
 ---
 module: ali_rds_instance
 version_added: "1.5.0"
-short_description: Create, Restart or Terminate an Instance in RDS.
+short_description: Create, Restart or Delete an RDS Instance.
 description:
-    - Create, restart, modify or terminate rds instances.
+    - Create, Restart, Delete, Create readonly instance and Modify network_type, connection_string, description, spec for RDS Instance.
 options:
   state:
     description:
       - The state of the instance after operating.
+    choices: ['present', 'restart', 'absent']
```

Comment:
```
  state:
    description:
        - If I(state=present), instance will be created.
        - If I(state=present) and instance exists, it will be updated.
        - If I(state=absent), instance will be removed.
        - If I(state=restart), instance will be restarted.
    default: present
    choices: ['present', 'restart', 'absent']
```
以后所有的state都这样写，说清楚每个state分别是干什么的